web/src/tippyjs.js

```diff
@@ -452,20 +452,38 @@ export function initialize() {
     });
 
     delegate("body", {
-        target: "#show_all_private_messages",
```

Comment:
I don't think you need to use this sort of hardcoded list of elements. Our defaults already register the `get_tooltip_content` function for getting the tooltip content from `data-tooltip-template-id`. In fact, @amanagr can the `.message_control_button` `onShow` method be deleted in favor of using the default `data-tooltip-template-id`?

So the only thing you'd; be customizing is the delay; for that, how about we create a variant called `.tippy-zulip-delayed-tooltip` that is identical to `tippy-zulip-tooltip` except it adds delay `LONG_HOVER_DELAY` (which should replace the 500ms values you have in this PR)? Then we can just apply that class to these elements rather than having a fragile list of selectors in this file.